catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java

```diff
@@ -460,7 +456,7 @@ private CreateResponse createMetacards(CatalogFacade catalog, List<Metacard> lis
   }
 
   private Metacard readMetacard(File file) throws IngestException {
-    Metacard result = null;
+    Metacard result;
```

Comment:
❗️ By removing `null` you are making it so that at line 501 below when we return, the value is undefined if for example, `new FileInputStream()` throws an exception.